### PR TITLE
Add 1.36 update notification

### PR DESCRIPTION
**Pull request by WorldLanguages**
Edit: screenshot is outdated

image

### CODE REVIEW

**Comment by WorldLanguages** [comment]
cc @DNin01 @Samq64 @mxmou

sorry for the hiatus :-) we will be releasing very shortly, and probably once again on the start of March

File: _locales/en/messages.json [hunk 1]
```diff
@@ -242,15 +242,15 @@
   "extensionHasUpdated": {
     "message": "Scratch Addons has updated to v$1"
   },
-  "extensionUpdateInfo1_v1_35": {
-    "message": "New addon: \"skewing in costume editor\", which lets you hold Ctrl while scaling an object on its sides to skew it. Go to $1 to enable it.",
+  "extensionUpdateInfo1_v1_36": {
+    "message": "New addon: \"place backpack code at mouse\", which lets you drag-and-drop code blocks to the mouse position instead of to the code area corner. Go to $1 to enable it.",
```
Comment (Samq64) [inline _locales/en/messages.json:246]: Coukd one of the other addons go here? This one is already enabled by default and doesn't sound very exciting.

Comment (WorldLanguages) [reply _locales/en/messages.json:246]: Which one would you highlight instead?
Personally I think it's fine to highlight an addon even if we already turned it on automatically.

Comment (Samq64) [reply _locales/en/messages.json:246]: I was thinking rearrangeable custom block inputs but any of them are fine.

Comment (Joeclinton1) [reply _locales/en/messages.json:246]: Was going to say that "place backpack code at mouse" is enabled by default. So the message doesn't make sense.

Comment (DNin01) [reply _locales/en/messages.json:246]: Furthermore, using Place Backpack Code at Mouse creates a very subtle change - the user may not be thinking about it, or even know that there was undesired behavior, nor would they have anything to say, most likely. It's kind of like how Faster Project Loading is enabled by default and no one really notices that anything is happening unless they read the addon's description.

**Pull request merged**